domain/model/state/types.go

```diff
@@ -410,3 +416,12 @@ type dbModelSecretBackend struct {
 	// SecretBackendUUID is the secret backends unique identifier.
 	SecretBackendUUID string `db:"secret_backend_uuid"`
 }
+
+// dbModelState is used to represent a single row from the
+// v_model_status_state view. This information is used to feed a model's status.
+type dbModelState struct {
+	Destroying              bool   `db:"destroying"`
```

Comment:
I wonder whether our convention for naming bool attributes should be `IsDestroying` etc.

Reply:
I am happy with either. Consistency!

Reply:
We haven't used this convention in the DB yet.